prow/istio-presubmit.sh

```diff
@@ -87,5 +87,3 @@ HUB="gcr.io/istio-testing"
 TAG="${GIT_SHA}"
 # upload images
 time make push HUB="${HUB}" TAG="${TAG}"
-
-time cd ${ROOT}/pilot; make e2etest HUB="${HUB}" TAG="${TAG}" TESTOPTS="-mixer=false"
```

Comment:
Why remove this ? We do want pilot to be part of the presubmit right ? Who can approve this change ? @mandarjog